CHANGELOG.md

```diff
@@ -7,6 +7,17 @@ Our versioning scheme is `YEAR.N` where `N` is incremented whenever a new releas
 ## [??? (unreleased)]
 
 
+## [2020.7] - 2020-05-29
+
+### Changed
+
+- Cache device data (network, hardware info...) as much as we can to save bandwidth [PR](https://github.com/advancedtelematic/aktualizr/pull/1673)
+
+### Fixed
+
+- Various docker-app fixes: [PR](https://github.com/advancedtelematic/aktualizr/pull/1664) [PR](https://github.com/advancedtelematic/aktualizr/pull/1665)
+- Use ED25519 to sign manifests when set as key type: [PR](https://github.com/advancedtelematic/aktualizr/pull/1608)
+
```

Comment:
Maybe add something about changing the installed Target matching logic from https://github.com/advancedtelematic/aktualizr/pull/1666?

Reply:
Do you have a suggestion to formulate this change? "Stricter checks for uptane metadata consistency"?

Reply:
How about "Stricter matching of Uptane metadata with installed images."?

Although to be honest, it's just that the strict check was moved from the install step to the check for updates.

Reply:
Ok changed it to yours